src/utilities/utilities.ts

```diff
@@ -62,6 +65,9 @@ export async function execFileStreamOutput(
         `Exec: ${executable} ${args.join(" ")}`,
         folderContext.name
     );
+    if (configuration.runtimePath.length > 0) {
+        options.env = { ...options.env, ...swiftRuntimePathEnv(options.env) };
+    }
```

Comment:
`options.env` may be undefined here, if that is the case you should use `process.env` as the basis for runtime path construction

Reply:
The same as last one.